app/views/orga/teams/_form.html.slim

```diff
@@ -76,32 +76,19 @@
   - if admin?
     = f.input :season_id, as: :select, collection: Season.order('name DESC') { |e| [s.name, s.id] }
 
+    == render 'teams/conference_preference_fields', f: f
```

Comment:
Can you please also make this a single `=` _(even though it maybe accidentally works, it just does not look semantically correct)_